backend/src/services/certificate-authority/certificate-authority-service.ts

```diff
@@ -384,14 +439,98 @@ export const certificateAuthorityServiceFactory = ({
       return castDbEntryToAcmeCertificateAuthority(certificateAuthority);
     }
 
+    if (type === CaType.AZURE_AD_CS) {
+      return castDbEntryToAzureAdCsCertificateAuthority(certificateAuthority);
+    }
+
     throw new BadRequestError({ message: "Invalid certificate authority type" });
   };
 
+  const orderSubscriberCertificate = async (subscriberId: string, actor: OrgServiceActor) => {
+    const subscriber = await pkiSubscriberDAL.findById(subscriberId);
+
+    if (!subscriber.caId) {
+      throw new BadRequestError({ message: "Subscriber does not have a CA" });
+    }
+
+    const ca = await certificateAuthorityDAL.findByIdWithAssociatedCa(subscriber.caId);
+
+    const { permission } = await permissionService.getProjectPermission({
+      actor: actor.type,
```

Comment:
are we using this anywhere? 

asking because ACME actually has to happen in the queue because it takes a long time to process so this wouldn't be applicable... 

also I don't see this being used but maybe I just missed it